asf_heat_pump_suitability/pipeline/transform/heat_network_zones.py

```diff
@@ -0,0 +1,42 @@
+"""
+Functions to filter OS UPRNs to UPRNs located within heat network zones.
+"""
+
+import geopandas as gpd
+
+from asf_heat_pump_suitability import config
+
+
+def filter_gdf_heat_network_zone_uprns(
+    uprn_gdf: gpd.GeoDataFrame,
+    hn_zone_gdf: gpd.GeoDataFrame,
+    hn_zone_usecols: list = None,
```

Comment:
```suggestion
    usecols: list = None,
```
I'd rename this for readability. Also, it needs to be added to docstring :)